cmd/api-server/main.go

```diff
@@ -192,10 +193,10 @@ func main() {
 	}
 
 	if clusterId == "" {
-		cmConfig, err = configRepository.Get(ctx)
+		cmConfig, _ = configRepository.Get(ctx)
```

Comment:
why do hide error there?

Reply:
it wasn't checked before, I just hide it because staticcheck was linting it for not being checked.

SHould it be checked here?

Reply:
Might be log for debug mode